libraries/resource_systemd_automount.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'resource_systemd_unit'
+require_relative 'systemd_automount'
+
+class Chef::Resource
+  class SystemdAutomount < Chef::Resource::SystemdUnit
+    self.resource_name = :systemd_automount
```

Comment:
hmmmmm... i lifted it from the [docker_service resource](https://github.com/bflad/chef-docker/blob/master/libraries/resource_docker_service.rb#L7), i'll dig a little deeper and see if it can be avoided.